src/Sulu/Bundle/MediaBundle/Media/Manager/MediaManager.php

```diff
@@ -643,7 +643,7 @@ protected function setDataToMedia(Media $media, $data, $user)
                         $media->removeTags();
                         if (\count($value)) {
                             foreach ($value as $tag) {
-                                $tagEntity = $this->tagManager->findOrCreateByName($tag, $user->getId());
```

Comment:
@alexander-schranz I think the `$user` is not even needed anymore in this method, can we completely remove it?

Or do you want to preserve it because of BC break and the method is `protected` ?